### PR TITLE
Wiki archive: exclude unneeded directories

### PR DESCRIPTION
**Pull request by Pegasus-RPG**
- use bzip2
- preserve permissions

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -111,8 +111,9 @@ def snapshot(snapshot_name=None):
             FORUMS_DATABASE_NAME, mysql_snapshot))
 
         wiki_path = os.path.join(instance_path, 'wiki')
-        wiki_snapshot = 'wiki.tar.gz'
-        sudo('tar czf {} -C {} .'.format(wiki_snapshot, wiki_path))
+        wiki_snapshot = 'wiki.tar.bz2'
+        # Exclusions to save space & time per https://www.dokuwiki.org/faq:backup
+        sudo('tar --exclude='data/cache' --exclude='data/index' --exclude='data/locks' --exclude='data/tmp' -cjpf {} -C {} .'.format(wiki_snapshot, wiki_path))
```
Comment (rryan) [inline fabfile.py]: hm, the single quotes need escaping

`'tar --exclude=\'data/cache\' --exclude=\'data/index\' --exclude=\'data/locks\' --exclude=\'data/tmp\' -cjpf {} -C {} .'`

**Comment by rryan** [comment]
thanks!

**Comment by rryan** [comment]
BTW you made a branch on mixxxdj for this (`Website-backup`). Please use your own fork for topic branches :)

**Comment by Pegasus-RPG** [comment]
Oh, I just edited the file right on GitHub's site. (MUCH easier than fussing with Git commands for such a small change.) It offered the option to create a new branch, so I did, not realizing it would do it as a sub-branch of MixxxDJ instead of my own, sorry.

**Pull request merged**